docs/concepts/glossary.md

```diff
@@ -0,0 +1,169 @@
+supervising the execution of the [Tasks](./glossary#task) of a single job.
+
+#### Logical Graph
+
+A logical graph is a directed graph describing the high-level logic of a stream processing program.
+The nodes are [Operators](./glossary#operator) and the edges indicate input/output-relationships or
+data streams or data sets.
+
+#### Managed State
+
+Managed State describes application state which has been registered with the framework. For
+Managed State, Apache Flink will take care about persistence and rescaling among other things.
+
+#### Flink JobManager Process
+
+The JobManager Process is the master of a [Flink Cluster](./glossary#flink-cluster). It is called
+*JobManager* for historical reasons, but actually contains three distinct components:
+Flink Resource Manager, Flink Dispatcher and one [Flink JobManager](./glossary#flink-jobmanager)
+per running [Flink Job](./glossary#flink-job).
+
+#### Operator
+
+Node of a [Logical Graph](./glossary#logical-graph). An Operator performs a certain operation,
+which is usually executed by a [Function](./glossary#function). Sources and Sinks are special
+Operators for data ingestion and data egress.
+
+#### Operator Chain
+
+An Operator Chain consists of two or more consecutive [Operators](./glossary#operator) without any
+repartitioning in between. Operators within the same Operation Chain forward records to each other
+directly without going through serialization or Flink's network stack.
+
+#### Partition
+
+A partition is an independent subset of the overall data stream or data set. A data stream or
+data set is divided into partitions by assigning each [record](./glossary#Record) to one or more
+partitions. Partitions of data streams or data sets are consumed by [Tasks](./glossary#task) during
+runtime. A transformation which changes the way a data stream or data set is partitioned is often
+called repartitioning.
+
+#### Physical Graph
+
+A physical graph is the result of translating a [Logical Graph](./glossary#logical-graph) for
+execution in a distributed runtime. The nodes are [Tasks](./glossary#task) and the edges indicate
+input/output-relationships or [partitions](./glossary#partition) of data streams or data sets.
+
+#### Record
+
+Records are the constituent elements of a data set or data stream.
+[Operators](./glossary#operator) and [Functions](./glossary#Function) receive records as input
+and emit records as output.
+
+#### Flink Session Cluster
+
+A long-running [Flink Cluster](./glossary#flink-cluster) which accepts multiple
+[Flink Jobs](./glossary#flink-job) for execution. The lifetime of this Flink Cluster is not bound
+to the lifetime of any Flink Job. Formerly, a Flink Session Cluster was also known as a Flink Cluster in
+*session mode*. Compare to [Flink Application Cluster](./glossary#flink-application-cluster).
+
+#### State Backend
+
+For stream processing programs, the State Backend determines how state is stored on each TaskManager
```

Comment:
`the State Backend determines how state is stored` -> `a State Backend determines how Flink Job's state is stored`?

In general, it's a little bit not clear from here, that a state backend can be configured on Flink job level.

Reply:
Should be clearer now.